src/main/kotlin/cloud/luxor/lbwl/flash/Util.kt

```diff
@@ -36,17 +37,18 @@ fun createTwitterLink(click: String, tweet: String, vararg hashtags: String): Ar
             }
         }
     }
-    val builder = ComponentBuilder(" $click")
-        .event(ClickEvent(ClickEvent.Action.OPEN_URL, url.toString().replace(" ", "%20")))
-        .event(HoverEvent(HoverEvent.Action.SHOW_TEXT, TextComponent.fromLegacyText("§7Öffnet ein Fenster zu Twitter.")));
-    return builder.create();
+    return Component.text(" $click").clickEvent(
+        net.kyori.adventure.text.event.ClickEvent.clickEvent(net.kyori.adventure.text.event.ClickEvent.Action.OPEN_URL, url.toString().replace(" ", "%20"))
```

Comment:
Why are you not importing these classes and instead use their FQCN?

Reply:
see [https://github.com/Freggy/lbwl-flash/pull/5/commits/c2c4cdf7a4242766f88c85d81114b3996b0d3361](https://github.com/Freggy/lbwl-flash/pull/5/commits/c2c4cdf7a4242766f88c85d81114b3996b0d3361)